# Ubuntu-ja translation of update-manager.
# Copyright (C) 2006 THE update-manager'S COPYRIGHT HOLDER
# This file is distributed under the same license as the update-manager package.
# Ikuya Awashiro <ikuya@fruitsbasket.info>, 2006.
# Hiroyuki Ikezoe <ikezoe@good-day.co.jp>, 2005
#
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager 0.42.4\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2009-03-28 11:40+0000\n"
"PO-Revision-Date: 2009-03-27 21:19+0000\n"
"Last-Translator: Shushi Kurose <md81bird@hitaki.net>\n"
"Language-Team: Ubuntu Japanese Team <ubuntu-ja-users@freeml.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2009-03-31 14:45+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:233 ../UpdateManager/Core/utils.py:233
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:236 ../UpdateManager/Core/utils.py:236
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:239 ../UpdateManager/Core/utils.py:239
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:242 ../UpdateManager/Core/utils.py:242
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:207 ../DistUpgrade/distro.py:422
#, python-format
msgid "Server for %s"
msgstr "%s のサーバ"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:225 ../DistUpgrade/distro.py:231
#: ../DistUpgrade/distro.py:247
msgid "Main server"
msgstr "メインサーバ"

#: ../DistUpgrade/distro.py:251
msgid "Custom servers"
msgstr "カスタムサーバ"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "ソースを計算できませんでした、エントリーのリストを作ってください"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr "パッケージファイルが見つかりません、Ubuntu Discでないかアーキテクチャーが間違っています"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "CDの追加に失敗しました"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD の追加に失敗したため、アップグレードは終了されます。この CD が正規の Ubuntu CD の場合は、このことをバグとして報告してください。\n"
"\n"
"エラーメッセージ：\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "壊れた状態のパッケージを削除する"

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"パッケージ '%s' は 矛盾した状態にあり再インストールが必要ですが、そのためのアーカイブを見つけることができません。 "
"このパッケージを削除して先に進めますか？"

#: ../DistUpgrade/DistUpgradeCache.py:330
msgid "Broken packages"
msgstr "壊れたパッケージ"

#: ../DistUpgrade/DistUpgradeCache.py:331
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"システムにはこのソフトウェアでは修復できない壊れたパッケージが含まれています。 先に進む前に、\r\n"
"まず Synaptic や apt-get を使って問題を修正してください。"

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:564
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"アップグレードの計算中に解決不能の問題が発生しました：\n"
"%s\n"
"\n"
" 以下の原因が考えられます：\n"
" * Ubuntuのプリリリースバージョンへアップグレードしようとしています。\n"
" * Ubuntuの現時点でリリースされていないバージョンを使っています。\n"
" * Ubuntuが提供していない私的ソフトウェアパッケージを利用しています。\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:574
msgid "This is most likely a transient problem, please try again later."
msgstr "これは一時的な問題のようです。後でもう一度試してください。"

#: ../DistUpgrade/DistUpgradeCache.py:577
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"これがどれも適用されないのであれば、'update-manager' パッケージに対するバグとして、/var/log/dist-upgrade/ "
"にあるファイルを含めて報告してください。"

#: ../DistUpgrade/DistUpgradeCache.py:583
#: ../UpdateManager/UpdateManager.py:717
msgid "Could not calculate the upgrade"
msgstr "アップグレードが算定できません"

#: ../DistUpgrade/DistUpgradeCache.py:633
msgid "Error authenticating some packages"
msgstr "いくつかのパッケージが認証されませんでした"

#: ../DistUpgrade/DistUpgradeCache.py:634
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"いくつかのパッケージが認証できませんでした。これは一時的なネットワークの問題かもしれません。あとで再び試してみてください。下に認証できなかったパッケージが"
"表示されます。"

#: ../DistUpgrade/DistUpgradeCache.py:653
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "パッケージ'%s'が削除に指定されましたが、削除ブラックリストに入っています。"

#: ../DistUpgrade/DistUpgradeCache.py:657
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "必須パッケージ'%s'に削除マークがついています。"

#: ../DistUpgrade/DistUpgradeCache.py:735
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "'%s'をアップグレードに指定できません。"

#: ../DistUpgrade/DistUpgradeCache.py:749
#, python-format
msgid "Can't install '%s'"
msgstr "'%s'をインストールすることができません。"

#: ../DistUpgrade/DistUpgradeCache.py:750
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr "要求されたパッケージのインストールができません。バグとして報告してください。 "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:758
msgid "Can't guess meta-package"
msgstr "メタパッケージを推測できません"

#: ../DistUpgrade/DistUpgradeCache.py:759
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"システムに ubuntu-desktop, kubuntu-desktop, xubuntu-desktop, もしくはedubuntu-desktop "
"パッケージが含まれていません。また、実行している ubuntu のバージョンが検出できません。 \n"
" 開始前に Synaptic や apt-get を使用して、まずは上記のパッケージのうちのひとつをインストールしてください。"

#: ../DistUpgrade/DistUpgradeController.py:85
msgid "Reading cache"
msgstr "キャッシュを読み込み中です"

#: ../DistUpgrade/DistUpgradeController.py:178
msgid "Unable to get exclusive lock"
msgstr "排他的なロックができません"

#: ../DistUpgrade/DistUpgradeController.py:179
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"パッケージマネージャは同時にひとつしか起動できません。apt-getやaptitudeなどの、他のパッケージマネージャを終了してください。"

#: ../DistUpgrade/DistUpgradeController.py:232
msgid "Upgrading over remote connection not supported"
msgstr "リモート接続を利用したアップグレードはサポートされていません"

#: ../DistUpgrade/DistUpgradeController.py:233
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. The upgrade will abort now. Please try without "
"ssh."
msgstr ""
"sshをサポートしていないフロントエンドで、sshのリモート接続をした状態でアップグレードを実行しています。アップグレードは中止します。sshを使わずに再"
"度試してください。"

#: ../DistUpgrade/DistUpgradeController.py:244
msgid "Continue running under SSH?"
msgstr "SSH越しに実行していますが、続けますか？"

#: ../DistUpgrade/DistUpgradeController.py:245
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"このセッションはSSH越しに実行されています。今のところ、失敗した場合の復旧が困難になるため、SSH越しのアップグレードは推奨されません。\n"
"\n"
"続ける場合、予備のSSHデーモンがポート'%s' で実行されます。\n"
"本当に続けますか？"

#: ../DistUpgrade/DistUpgradeController.py:260
msgid "Starting additional sshd"
msgstr "予備のsshdを起動しています"

#: ../DistUpgrade/DistUpgradeController.py:261
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"障害が起こったときに復旧しやすくするため、ポート '%s' でもう一つの sshd を開始します。ssh "
"の実行がうまくいかなくても、もう一方のポートに接続することができます。\n"

#: ../DistUpgrade/DistUpgradeController.py:331
#: ../DistUpgrade/DistUpgradeController.py:376
msgid "Can not upgrade"
msgstr "アップグレードできません"

#: ../DistUpgrade/DistUpgradeController.py:332
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "'%s' から '%s' へのアップグレードは、このツールではサポートされていません。"

#: ../DistUpgrade/DistUpgradeController.py:341
msgid "Sandbox setup failed"
msgstr "サンドボックスのセットアップに失敗しました"

#: ../DistUpgrade/DistUpgradeController.py:342
msgid "It was not possible to create the sandbox environment."
msgstr "サンドボックス環境を作成することができませんでした。"

#: ../DistUpgrade/DistUpgradeController.py:348
msgid "Sandbox mode"
msgstr "サンドボックスモード"

#: ../DistUpgrade/DistUpgradeController.py:349
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:377
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr "インストールされたPythonが破損しています。シンボリックリンク'/usr/bin/python'を修正してください。"

#: ../DistUpgrade/DistUpgradeController.py:403
msgid "Package 'debsig-verify' is installed"
msgstr "'debsig-verify'パッケージがインストールされています"

#: ../DistUpgrade/DistUpgradeController.py:404
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"'debsig-verify'パッケージがインストールされていると、アップグレードを継続することができません。\n"
"Synapticパッケージ・マネージャか 'apt-get remove debsig-verify' "
"で削除した後で、再度アップグレードを実行してください。"

#: ../DistUpgrade/DistUpgradeController.py:414
msgid "Include latest updates from the Internet?"
msgstr "インターネットからの最新のアップデートを取得しますか？"

#: ../DistUpgrade/DistUpgradeController.py:415
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"インターネットを使ってシステムをアップグレードすることもできます。その場合、最新のパッケージが自動的にダウンロードされインストールされますので、もしネット"
"ワーク接続が可能ならネットワークアップグレードを強く推奨します。\n"
"ネットワークアップグレードは時間がかかるかもしれませんが、完了したら完全に最新の状態に更新されます。ネットワークを利用しないアップグレードも可能ですが、そ"
"の場合はアップグレード完了後、さらに最新の状態を取得する必要があります。\n"
"ここで「いいえ」を選択すると、アップグレード中はネットワークを利用しません。"

#: ../DistUpgrade/DistUpgradeController.py:626
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%sへのアップグレードは無効化されました"

#: ../DistUpgrade/DistUpgradeController.py:652
msgid "No valid mirror found"
msgstr "有効なミラーが見つかりません"

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found.This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'no' the update will cancel."
msgstr ""
"リポジトリ情報をスキャンしましたが、アップグレードのためのミラーエントリが見つかりませんでした。内部ミラーもしくはミラー情報が古い場合に起こることがありま"
"す。\n"
"\n"
"'sources.list' ファイルを書き換えますか？　'はい' を選択すると '%s' エントリを '%s' エントリにアップデートします。 "
"'いいえ' を選択するとアップデートをキャンセルします。"

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:673
msgid "Generate default sources?"
msgstr "標準のソースを生成しますか？"

#: ../DistUpgrade/DistUpgradeController.py:674
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the update "
"will cancel."
msgstr ""
"'sources.list' を読み込みましたが、 '%s' に対する有効なエントリが見つかりませんでした。\n"
"\n"
"'%s' に対するデフォルトのエントリを追加しますか？ 'いいえ' を選択するとアップデートを中断します。"

#: ../DistUpgrade/DistUpgradeController.py:709
msgid "Repository information invalid"
msgstr "リポジトリ情報が無効です"

#: ../DistUpgrade/DistUpgradeController.py:710
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr "リポジトリ情報をアップグレード中に無効なファイルを生成しました。バグとして報告してください。"

#: ../DistUpgrade/DistUpgradeController.py:716
msgid "Third party sources disabled"
msgstr "公式ではないソースが無効になりました"

#: ../DistUpgrade/DistUpgradeController.py:717
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"sources.list にある公式ではないエントリを無効にしました。アップグレード後、'ソフトウェア・ソース' "
"ツールもしくはパッケージ・マネージャを使って再び有効にすることができます。"

#: ../DistUpgrade/DistUpgradeController.py:758
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "矛盾した状態のパッケージ"

#: ../DistUpgrade/DistUpgradeController.py:761
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"パッケージ '%s' は "
"矛盾した状態にあり再インストールが必要ですが、そのためのアーカイブを見つけることができません。パッケージを手動で再インストールするか、システムからパッケー"
"ジを削除してください。"

#: ../DistUpgrade/DistUpgradeController.py:809
msgid "Error during update"
msgstr "アップデート中にエラー発生"

#: ../DistUpgrade/DistUpgradeController.py:810
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"アップデート中に問題が発生しました。通常、これはある種のネットワーク問題です。ネットワーク接続をチェックし、再びアップデートしてください。"

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:819
#: ../UpdateManager/UpdateManager.py:553
msgid "Not enough free disk space"
msgstr "ディスクの空き領域が足りません"

#: ../DistUpgrade/DistUpgradeController.py:820
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:860
msgid "Support for some applications ended"
msgstr "いくつかのアプリケーションのサポートは終了しました"

#: ../DistUpgrade/DistUpgradeController.py:861
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""
"Canonical Ltd. は以下のソフトウェアパッケージをもうサポートしておらず、コミュニティサポートになっています。\n"
"\n"
"コミュニティ管理のソフトウェア(universe)を有効にしていなければ、これらのパッケージはアップグレードの最終段階で、削除するように提案されます。"

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:870
#: ../DistUpgrade/DistUpgradeController.py:1522
msgid "Calculating the changes"
msgstr "変更点を取得中"

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:885
msgid "Do you want to start the upgrade?"
msgstr "アップグレードを開始しますか？"

#: ../DistUpgrade/DistUpgradeController.py:937
#: ../DistUpgrade/DistUpgradeController.py:1009
msgid "Could not download the upgrades"
msgstr "アップグレードをダウンロードできません"

#: ../DistUpgrade/DistUpgradeController.py:938
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"現在アップグレードは中断されています。インターネット接続やインストールメディアを確認してもう一度試してください。これまでにダウンロードしたファイルはすべて"
"保存されています。"

#. invoke the frontend now
#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"現在アップグレードは中断されています。システムが使用できない状態である可能性があります。今から回復を実行します。 (dpkg --configure -"
"a)"

#: ../DistUpgrade/DistUpgradeController.py:989
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"このバグを 'update-manager' パッケージに対するバグとして、/var/log/dist-upgrade/ "
"にあるファイルをバグレポートに含めて報告してください。\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:993
msgid "Could not install the upgrades"
msgstr "アップグレードをインストールできません"

#: ../DistUpgrade/DistUpgradeController.py:1010
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr "現在アップグレードは中断されています。インターネット接続やインストールメディアを確認してもう一度試してください。 "

#: ../DistUpgrade/DistUpgradeController.py:1088
msgid "Remove obsolete packages?"
msgstr "不要なパッケージを削除しますか？"

#: ../DistUpgrade/DistUpgradeController.py:1089
#: ../DistUpgrade/DistUpgrade.glade.h:20
msgid "_Keep"
msgstr "そのまま(_K)"

#: ../DistUpgrade/DistUpgradeController.py:1089
msgid "_Remove"
msgstr "削除(_R)"

#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Error during commit"
msgstr "コミット中にエラーが発生しました"

#: ../DistUpgrade/DistUpgradeController.py:1101
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "クリーンアップ中に問題が発生しました。詳細情報は下記のメッセージを参照してください。 "

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1139
msgid "Restoring original system state"
msgstr "システムを元に戻しています"

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1177
msgid "Required depends is not installed"
msgstr "依存するパッケージがインストールされていません"

#: ../DistUpgrade/DistUpgradeController.py:1178
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "依存するパッケージ '%s' がインストールされていません。 "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1432
#: ../DistUpgrade/DistUpgradeController.py:1499
msgid "Checking package manager"
msgstr "パッケージ・マネージャをチェック中です"

#: ../DistUpgrade/DistUpgradeController.py:1437
msgid "Preparing the upgrade failed"
msgstr "アップグレードの準備に失敗しました"

#: ../DistUpgrade/DistUpgradeController.py:1438
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"アップグレードのためにシステムを準備している途中で失敗しました。このことを 'update-manager' "
"パッケージに対するバグとして、/var/log/dist-upgrade/ にあるファイルをバグレポートに含めて報告してください。"

#: ../DistUpgrade/DistUpgradeController.py:1454
msgid "Getting upgrade prerequisites failed"
msgstr "アップグレードの事前作業に失敗しました"

#: ../DistUpgrade/DistUpgradeController.py:1455
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1485
msgid "Updating repository information"
msgstr "リポジトリ情報をアップデート中です"

#: ../DistUpgrade/DistUpgradeController.py:1510
msgid "Invalid package information"
msgstr "無効なパッケージ情報です"

#: ../DistUpgrade/DistUpgradeController.py:1511
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1528
#: ../DistUpgrade/DistUpgradeController.py:1576
msgid "Fetching"
msgstr "取得中"

#: ../DistUpgrade/DistUpgradeController.py:1535
#: ../DistUpgrade/DistUpgradeController.py:1580
msgid "Upgrading"
msgstr "アップグレード中です"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1540
#: ../DistUpgrade/DistUpgradeController.py:1582
#: ../DistUpgrade/DistUpgradeController.py:1589
#: ../DistUpgrade/DistUpgradeController.py:1594
msgid "Upgrade complete"
msgstr "アップグレードが完了しました"

#: ../DistUpgrade/DistUpgradeController.py:1541
#: ../DistUpgrade/DistUpgradeController.py:1583
#: ../DistUpgrade/DistUpgradeController.py:1590
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr "アップグレードは完了しましたが、アップグレード処理中にエラーが発生しました。"

#: ../DistUpgrade/DistUpgradeController.py:1548
msgid "Searching for obsolete software"
msgstr "古いソフトウェアを検索しています"

#: ../DistUpgrade/DistUpgradeController.py:1553
msgid "System upgrade is complete."
msgstr "システムのアップグレードが完了しました。"

#: ../DistUpgrade/DistUpgradeController.py:1595
msgid "The partial upgrade was completed."
msgstr "部分的なアップグレードが完了しました。"

#: ../DistUpgrade/DistUpgradeQuirks.py:166
#: ../DistUpgrade/DistUpgradeQuirks.py:235
#: ../DistUpgrade/DistUpgradeQuirks.py:260
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr "アップグレードによってデスクトップ効果やゲームのグラフィックに強く依存したその他のプログラムのパフォーマンスが低下するかもしれません。"

#: ../DistUpgrade/DistUpgradeQuirks.py:170
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 9.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"このコンピューターは現在AMD'fglrx'のグラフィックドライバを使っています、Ubuntu9.04.には、あなたのハードウェアのドライバがありません。"

#: ../DistUpgrade/DistUpgradeQuirks.py:239
#: ../DistUpgrade/DistUpgradeQuirks.py:264
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 8.10.\n"
"\n"
"Do you want to continue?"
msgstr ""
"現在、このコンピュータはNVIDIA 'nvidia' グラフィックドライバを利用しています。このドライバの、Ubuntu "
"8.10用バージョンはまだ利用できません。\n"
"\n"
"それでも作業を続けますか？"

#: ../DistUpgrade/DistUpgradeQuirks.py:434
msgid "evms in use"
msgstr "evms は使用中です"

#: ../DistUpgrade/DistUpgradeQuirks.py:435
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"あなたのシステムは'evms' "
"をボリュームマネジャとして/proc/mountsで使っています。'evms'ソフトのサポートは終了しました。オフにしてからアップグレードして下さい。"

#: ../DistUpgrade/DistUpgradeMain.py:38
msgid "Sandbox upgrade using aufs"
msgstr "aufsを使用したサンドボックスアップグレード"

#: ../DistUpgrade/DistUpgradeMain.py:40
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "アップグレードパッケージが含まれたCD-ROMへのパスを"

#: ../DistUpgrade/DistUpgradeMain.py:46
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"フロントエンドを使用してください。現在利用可能なもの: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:49
msgid "*DEPRECATED* this option will be ignore"
msgstr "*廃止予定* このオプションは無視されます"

#: ../DistUpgrade/DistUpgradeMain.py:52
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "部分アップグレードだけを実行します（sources.listを書き換えません）"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "Set datadir"
msgstr "datadirを設定"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:108
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "ドライブ '%s' に'%s' を挿入してください"

#: ../DistUpgrade/DistUpgradeViewGtk.py:127
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "取得が完了しました"

#: ../DistUpgrade/DistUpgradeViewGtk.py:139
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li / %li を取得中（%s B/s）"

#: ../DistUpgrade/DistUpgradeViewGtk.py:140
#: ../DistUpgrade/DistUpgradeViewGtk.py:275
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:357
#, python-format
msgid "About %s remaining"
msgstr "残り時間 約%s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:142
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li / %li を取得中です"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:174
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "変更を適用しています"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:199
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "依存関係の問題 - 設定を見送ります"

#: ../DistUpgrade/DistUpgradeViewGtk.py:204
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "'%s' がインストールできません"

#: ../DistUpgrade/DistUpgradeViewGtk.py:205
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may be in a not working "
"state. Please consider submitting a bug report about it."
msgstr ""
"アップグレードを続けますが、'%s' パッケージが動かない状態である可能性があります。このことについてのバグレポートを送信することを検討してください。"

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:221
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"以下のカスタマイズ済みの設定ファイルを置き換えますか？\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:222
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr "より新しいバージョンと入れ換えた場合、この構成ファイルへの変更内容は失われます。"

#: ../DistUpgrade/DistUpgradeViewGtk.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:309
msgid "The 'diff' command was not found"
msgstr "'diff' コマンドが見つかりません"

#: ../DistUpgrade/DistUpgradeViewGtk.py:417
#: ../DistUpgrade/DistUpgradeViewText.py:107
msgid "A fatal error occurred"
msgstr "致命的なエラーが起こりました"

#: ../DistUpgrade/DistUpgradeViewGtk.py:418
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:433
msgid "Ctrl-c pressed"
msgstr "Ctrl-cキーが押されました"

#: ../DistUpgrade/DistUpgradeViewGtk.py:434
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr "これは操作を中断することになり、システムを壊れた状態にするかもしれません。本当に実行しますか？"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:570
msgid "To prevent data loss close all open applications and documents."
msgstr "データの損失を避けるため、すべてのアプリケーションとドキュメントを閉じてください。"

#: ../DistUpgrade/DistUpgradeViewGtk.py:584
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>ダウングレードされるパッケージ： %s</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:586
#: ../DistUpgrade/DistUpgradeViewKDE.py:750
#, python-format
msgid "Remove %s"
msgstr "%s を削除"

#: ../DistUpgrade/DistUpgradeViewGtk.py:591
#: ../DistUpgrade/DistUpgradeViewKDE.py:752
#, python-format
msgid "Install %s"
msgstr "インストール %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:593
#: ../DistUpgrade/DistUpgradeViewKDE.py:754
#, python-format
msgid "Upgrade %s"
msgstr "アップグレード %s"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "メディアの交換"

#: ../DistUpgrade/DistUpgradeViewKDE.py:321
msgid "Show Difference >>>"
msgstr "端末を表示 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:324
msgid "<<< Hide Difference"
msgstr "<<< 端末を隠す"

#: ../DistUpgrade/DistUpgradeViewKDE.py:531
msgid "Error"
msgstr "エラー"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "&Cancel"
msgstr "キャンセル(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:545
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
msgid "&Close"
msgstr "閉じる(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:591
msgid "Show Terminal >>>"
msgstr "端末を表示 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:594
msgid "<<< Hide Terminal"
msgstr "<<< 端末を隠す"

#: ../DistUpgrade/DistUpgradeViewKDE.py:674
msgid "Information"
msgstr "情報"

#: ../DistUpgrade/DistUpgradeViewKDE.py:724
#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#: ../DistUpgrade/DistUpgradeViewKDE.py:768
#: ../DistUpgrade/DistUpgrade.glade.h:9
msgid "Details"
msgstr "詳細情報"

#: ../DistUpgrade/DistUpgradeViewKDE.py:778
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "Restart required"
msgstr "再起動してください"

#: ../DistUpgrade/DistUpgradeViewKDE.py:778
#: ../DistUpgrade/DistUpgrade.glade.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>アップグレードを完了するためにシステムの再起動が必要です</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgrade.glade.h:23
msgid "_Restart Now"
msgstr "すぐに再起動(_R)"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:793
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>実行中のアップグレードをキャンセルしますか？</big></b>\n"
"\n"
"アップグレードをキャンセルすると、システムが不安定な状態になる可能性があります。アップグレードを再開することを強くお薦めします。"

#: ../DistUpgrade/DistUpgradeViewKDE.py:797
msgid "Cancel Upgrade?"
msgstr "アップグレードを中断しますか?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li 日"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li 時間"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li 分"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li 秒"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:92
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:113
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr "このダウンロードは 1Mbit DSL 接続で約 %s 、56k モデムで約 %s かかります。"

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:117
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "このダウンロードは約 %s かかります。 "

#: ../DistUpgrade/DistUpgradeView.py:207 ../DistUpgrade/DistUpgrade.glade.h:14
msgid "Preparing to upgrade"
msgstr "アップグレードの準備"

#: ../DistUpgrade/DistUpgradeView.py:208
msgid "Getting new software channels"
msgstr "新しいソフトウェア・チャンネルの取得"

#: ../DistUpgrade/DistUpgradeView.py:209 ../DistUpgrade/DistUpgrade.glade.h:12
msgid "Getting new packages"
msgstr "新しいパッケージの取得"

#: ../DistUpgrade/DistUpgradeView.py:210 ../DistUpgrade/DistUpgrade.glade.h:13
msgid "Installing the upgrades"
msgstr "アップグレードのインストール"

#: ../DistUpgrade/DistUpgradeView.py:211 ../DistUpgrade/DistUpgrade.glade.h:8
msgid "Cleaning up"
msgstr "クリーンアップ"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:284
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d 個のパッケージが削除されます。"

#: ../DistUpgrade/DistUpgradeView.py:289
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d 個の新規パッケージがインストールされます。"

#: ../DistUpgrade/DistUpgradeView.py:295
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d 個のパッケージがアップグレードされます。"

#: ../DistUpgrade/DistUpgradeView.py:300
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"全部で %s をダウンロードする必要があります。 "

#: ../DistUpgrade/DistUpgradeView.py:304
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr "アップグレードの取得とインストールには数時間かかることがあります。ダウンロードが終了すると、処理を中断することはできません。"

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:310 ../UpdateManager/UpdateManager.py:446
msgid "Your system is up-to-date"
msgstr "システムは最新の状態です"

#: ../DistUpgrade/DistUpgradeView.py:311
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "システムに有効なアップグレードはありません。アップグレードは中止されます。"

#: ../DistUpgrade/DistUpgradeView.py:324
msgid "Reboot required"
msgstr "再起動が必要です"

#: ../DistUpgrade/DistUpgradeView.py:325
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "アップグレードが終了しました。再起動が必要ですが、すぐに実行しますか？"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:134
#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:134
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid "Could not run the upgrade tool"
msgstr "アップグレード・ツールを実行できません"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr "これはおそらくアップグレード・ツールのバグです。報告してください。"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool signature"
msgstr "アップグレード・ツールの署名"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool"
msgstr "アップグレード・ツール"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "取得に失敗しました"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "アップグレードの取得に失敗しました。おそらくネットワークの問題です。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "抽出に失敗しました"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "アップグレードの展開に失敗しました。おそらくネットワークまたはサーバの問題です。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verfication failed"
msgstr "検証に失敗しました"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "アップグレードの検証に失敗しました。ネットワーク接続もしくはサーバに問題があるかもしれません。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "認証に失敗しました"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "アップグレードの認証に失敗しました。おそらくネットワークまたはサーバの問題です。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:263
#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:263
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Can not run the upgrade"
msgstr "アップグレードを実行できません"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:264
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:264
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"これは主に /tmp が noexec オプションでマウントされているシステムで起こります。noexec "
"なしで再マウントし、アップグレードを実行しなおしてください。"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
#, python-format
msgid "The error message is '%s'."
msgstr "エラーメッセージは'%s'です。"

#: ../DistUpgrade/DistUpgradeViewText.py:108
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"このことをバグとして、ファイル /var/log/dist-upgrade/main.log と /var/log/dist-"
"upgrade/apt.log をレポートに含めて報告してください。現在アップグレードは中断されています。\n"
"元の sources.list は /etc/apt/sources.list.distUpgrade に保存されています。"

#: ../DistUpgrade/DistUpgradeViewText.py:132
msgid "Aborting"
msgstr "中断しました"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "Demoted:\n"
msgstr "降格:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
#: ../DistUpgrade/DistUpgradeViewText.py:198
msgid "Continue [yN] "
msgstr "続行する(yN) "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
msgid "Details [d]"
msgstr "詳細 [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:173
#: ../DistUpgrade/DistUpgradeViewText.py:201
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:176
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:183
#, python-format
msgid "Remove: %s\n"
msgstr "削除: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, python-format
msgid "Install: %s\n"
msgstr "インストール: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:189
#, python-format
msgid "Upgrade: %s\n"
msgstr "アップグレード: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
msgid "Continue [Yn] "
msgstr "続ける [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:226
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"アップグレードを完了するには再起動が必要です。\n"
"'Y' を選択すると再起動します。"

#: ../DistUpgrade/DistUpgrade.glade.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.glade.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>アップグレードをキャンセルしますか？</big></b>\n"
"\n"
"アップグレードをキャンセルすると、おそらくシステムは不安定な状態になります。アップグレードを再開することを強くおすすめします。"

#: ../DistUpgrade/DistUpgrade.glade.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>アップグレードを開始しますか？</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:7
msgid "<b><big>Upgrading Ubuntu to version 9.04</big></b>"
msgstr "<b><big>Ubuntu をバージョン 9.04にアップグレードしています</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:10
msgid "Difference between the files"
msgstr "ファイル間の違い"

#: ../DistUpgrade/DistUpgrade.glade.h:11
msgid "Distribution Upgrade"
msgstr "ディストリビューションのアップグレード"

#: ../DistUpgrade/DistUpgrade.glade.h:15
msgid "Restarting the computer"
msgstr "コンピュータの再起動"

#: ../DistUpgrade/DistUpgrade.glade.h:16
msgid "Setting new software channels"
msgstr "新しいソフトウェア・チャンネルの設定"

#: ../DistUpgrade/DistUpgrade.glade.h:17
msgid "Terminal"
msgstr "端末"

#: ../DistUpgrade/DistUpgrade.glade.h:18
msgid "_Cancel Upgrade"
msgstr "アップグレードを中断(_C)"

#: ../DistUpgrade/DistUpgrade.glade.h:19
msgid "_Continue"
msgstr "続行する(_C)"

#: ../DistUpgrade/DistUpgrade.glade.h:21
msgid "_Replace"
msgstr "置き換える(_R)"

#: ../DistUpgrade/DistUpgrade.glade.h:22
msgid "_Report Bug"
msgstr "バグを報告する(_R)"

#: ../DistUpgrade/DistUpgrade.glade.h:24
msgid "_Resume Upgrade"
msgstr "アップグレードを再開(_R)"

#: ../DistUpgrade/DistUpgrade.glade.h:25
msgid "_Start Upgrade"
msgstr "アップグレードを開始(_S)"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "リリースノートが見つかりません"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "サーバに過負荷がかかっています。 "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "リリースノートをダウンロードできません"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "インターネット接続を確認してください。"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "アップグレード"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.glade.h:11
msgid "Release Notes"
msgstr "リリースノート"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "追加パッケージをダウンロードしています..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "ファイル %s / %s (%sB/s)"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "ファイル %s の %s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "ブラウザでリンクを開く"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "クリップボードにリンクをコピー"

#: ../UpdateManager/GtkProgress.py:108
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "ファイル %(current)li / %(total)li をダウンロード中 (%(speed)s/秒)"

#: ../UpdateManager/GtkProgress.py:113
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "ファイル %(current)li / %(total)li をダウンロード中"

#: ../UpdateManager/UpdateManager.py:264
#, python-format
msgid "Version %s: \n"
msgstr "バージョン %s: \n"

#: ../UpdateManager/UpdateManager.py:325
msgid "Downloading list of changes..."
msgstr "変更点のリストを取得中です..."

#: ../UpdateManager/UpdateManager.py:357
msgid "_Uncheck All"
msgstr "すべてのチェックをはずす(_U)"

#: ../UpdateManager/UpdateManager.py:363
msgid "_Check All"
msgstr "全てチェック(_C)"

#. TRANSLATORS: b stands for Bytes
#: ../UpdateManager/UpdateManager.py:407 ../UpdateManager/UpdateManager.py:467
#, python-format
msgid "Download size: %s"
msgstr "ダウンロードサイズ: %s"

#: ../UpdateManager/UpdateManager.py:411
msgid "Unknown download size"
msgstr "不明なダウンロードサイズ"

#: ../UpdateManager/UpdateManager.py:419
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr "最後にパッケージ情報が更新された日時が不明です。 '確認' ボタンをクリックしてパッケージ情報を更新してください。"

#: ../UpdateManager/UpdateManager.py:428
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "パッケージ情報は %s 日前にアップデートされました。"

#: ../UpdateManager/UpdateManager.py:432
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "パッケージ情報は %s 時間前にアップデートされました。"

#: ../UpdateManager/UpdateManager.py:436
msgid "The package information was last updated less than one hour ago."
msgstr "パッケージ情報は1時間以内に更新されました"

#: ../UpdateManager/UpdateManager.py:444
#: ../data/glade/UpdateManager.glade.h:21
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr "ソフトウェアアップデートはエラーを訂正し、セキュリティホールを修正し、新機能を提供します。"

#: ../UpdateManager/UpdateManager.py:461
msgid "Welcome to Ubuntu"
msgstr "Ubuntuへようこそ"

#: ../UpdateManager/UpdateManager.py:462
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"このソフトウェアアップデートは、Ubuntuがリリースされて以降に公表されたものです。今すぐにインストールしたくない場合は、あとでシステム管理メニューから"
"“アップデート・マネージャ”を選択してください。"

#: ../UpdateManager/UpdateManager.py:465
msgid "Software updates are available for this computer"
msgstr "このコンピュータに対するソフトウェアアップデートがあります"

#: ../UpdateManager/UpdateManager.py:466
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr "今すぐにインストールしたくない場合は、あとでシステム管理メニューから“アップデート・マネージャ”を選択してください。"

#: ../UpdateManager/UpdateManager.py:505
msgid "Please wait, this can take some time."
msgstr "しばらくお待ちください。少々時間がかかります。"

#: ../UpdateManager/UpdateManager.py:507
msgid "Update is complete"
msgstr "アップデートが完了しました"

#: ../UpdateManager/UpdateManager.py:554
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"アップグレードには全体で%sの空き領域がディスク'%s'に必要です。少なくとも%sの領域を'%s'に作って下さい。ゴミ箱を空にしたり、'sudo apt-"
"get clean'を使って以前のインストールで利用した一時パッケージを削除して下さい。"

#: ../UpdateManager/UpdateManager.py:593
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "パッケージ情報を取得中"

#: ../UpdateManager/UpdateManager.py:695
msgid "Could not initialize the package information"
msgstr "パッケージ情報を初期化できませんでした"

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"パッケージ情報を初期化している最中に解決できない問題が発生しました．\n"
"\n"
"次のエラーメッセージを含めて'update-manager'パッケージに対して次のバグを報告してください:\n"

#: ../UpdateManager/UpdateManager.py:718
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"アップグレードの計算中に解決できない問題が発生しました．\n"
"\n"
"次のエラーメッセージを含めて'update-manager'パッケージに対して次のバグを報告してください:"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:743
#, python-format
msgid "(Size: %s)"
msgstr "(サイズ: %s)"

#: ../UpdateManager/UpdateManager.py:745
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "バージョン %(old_version)s から %(new_version)s　へ"

#: ../UpdateManager/UpdateManager.py:749
#, python-format
msgid "Version %s"
msgstr "バージョン %s"

#: ../UpdateManager/UpdateManager.py:762
msgid "Your distribution is not supported anymore"
msgstr "このディストリビューションはすでにサポート対象外です"

#: ../UpdateManager/UpdateManager.py:763
msgid ""
"You will not get any further security fixes or critical updates. Upgrade to "
"a later version of Ubuntu Linux. See http://www.ubuntu.com for more "
"information on upgrading."
msgstr ""
"セキュリティフィックスやアップデートはもう提供されません。最新バージョンの Ubuntu Linux にアップグレードしてください。\r\n"
"アップグレードに関する詳細な情報は http://www.ubuntu.com をご覧ください。"

#: ../UpdateManager/UpdateManager.py:795
#, python-format
msgid "<b>New distribution release '%s' is available</b>"
msgstr "<b>新しいディストリビューション '%s' にアップグレードできます</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:834
msgid "Software index is broken"
msgstr "ソフトウェアのインデックスが壊れています"

#: ../UpdateManager/UpdateManager.py:835
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"何らかのソフトウェアがインストールないし削除できません。 Synaptic パッケージ・マネージャを使用するか、 まずこの問題を解決するために "
"\"sudo apt-get install -f\" コマンドをターミナルで実行してください。"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "パッケージ情報をダウンロードしています"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "パッケージのアップグレードをダウンロードしています。"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr "選択したアップグレード・パッケージのダウンロードとインストールをしています。"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "run_synaptic が不明なアクションとともに呼び出されました"

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "キャンセル"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "インストール"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "アップデートリストを作成しています"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Problems with some of the installed software\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"正規のアップグレードを計算できません、次のものを起動してください: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"原因と考えられるものは:\n"
" ◦前回のアップグレードを終了していなかった、\n"
" ◦ インストールしたソフトウェアのどれかに問題がある\n"
" ◦ Ubuntuの提供していない私的なソフトウェア\n"
" ◦ Ubuntuのリリース前バージョンの正規の変更"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "変更履歴をダウンロードしています"

#: ../UpdateManager/Core/MyCache.py:209
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"変更履歴には関連する変更点がありません。\n"
"\n"
"変更点が表示されるようになるまで、http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"を利用するか、後ほど再び試してください。"

#: ../UpdateManager/Core/MyCache.py:214
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"変更点のリストはまだ利用できません。\n"
"\n"
"変更点が表示されるようになるまで、http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"を利用するか、後ほど再び試してください。"

#: ../UpdateManager/Core/MyCache.py:220
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr "変更点のダウンロードに失敗しました。インターネットに接続されているか確認してください。"

#: ../UpdateManager/Core/UpdateList.py:47
msgid "Failed to detect distribution"
msgstr "ディストリビューションの検出に失敗しました"

#: ../UpdateManager/Core/UpdateList.py:48
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "あなたが何のシステムを使用しているかについて確認している最中にエラー '%s' が起きました。"

#: ../UpdateManager/Core/UpdateList.py:59
msgid "Important security updates"
msgstr "重要なセキュリティアップデート"

#: ../UpdateManager/Core/UpdateList.py:60
msgid "Recommended updates"
msgstr "推奨アップデート"

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Proposed updates"
msgstr "提案されたアップデート"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Backports"
msgstr "バックポート"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Distribution updates"
msgstr "ディストリビューションのアップデート"

#: ../UpdateManager/Core/UpdateList.py:68
msgid "Other updates"
msgstr "その他のアップデート"

#: ../UpdateManager/Core/UpdateList.py:77
msgid "Previous selected"
msgstr "以前に選択したもの"

#: ../data/glade/UpdateManager.glade.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>アップデートの情報を手動でチェックしてください</big></b>\n"
"\n"
"システムはアップデートを自動的にチェックしません。この動作の変更は、<i>ソフトウェア・ソース</i>の<i>インターネットアップデート</i>タブで行い"
"ます。"

#: ../data/glade/UpdateManager.glade.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>一部のアップデートだけがインストールされました</b></big>"

#: ../data/glade/UpdateManager.glade.h:5
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>アップデート・マネージャを開始しています</b></big>"

#: ../data/glade/UpdateManager.glade.h:6
msgid "Changes"
msgstr "変更"

#: ../data/glade/UpdateManager.glade.h:7
msgid "Chec_k"
msgstr "再チェック(_K)"

#: ../data/glade/UpdateManager.glade.h:8
msgid "Check the software channels for new updates"
msgstr "新しいアップデートの調査のためにソフトウェア・チャンネルをチェックします"

#: ../data/glade/UpdateManager.glade.h:9
msgid "Description"
msgstr "詳細"

#: ../data/glade/UpdateManager.glade.h:10
msgid "Description of update"
msgstr "アップデートの詳細"

#: ../data/glade/UpdateManager.glade.h:12
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Problems with some of the installed software\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""
"部分的なアップグレードを行うと、可能な範囲でアップデートが実行されます。\n"
"\n"
"これは以下のような原因があります。\n"
"・以前のアップグレードが完了していません。\n"
"・Ubuntuで提供していない非公式パッケージ。\n"
"・Ubuntuのリリース前バージョンでは、これが通常の更新です。"

#: ../data/glade/UpdateManager.glade.h:19
msgid "Settings…"
msgstr "設定中..."

#: ../data/glade/UpdateManager.glade.h:20
msgid "Show progress of individual files"
msgstr "個々のファイルの進捗を表示"

#: ../data/glade/UpdateManager.glade.h:22
msgid "U_pgrade"
msgstr "アップグレード(_P)"

#: ../data/glade/UpdateManager.glade.h:23
#: ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "アップデート・マネージャ"

#: ../data/glade/UpdateManager.glade.h:24
msgid "Upgrade to the latest version of Ubuntu"
msgstr "Ubuntu の最新バージョンにアップグレード"

#: ../data/glade/UpdateManager.glade.h:25
msgid "_Hide this information in the future"
msgstr "今後この情報を隠す(_H)"

#: ../data/glade/UpdateManager.glade.h:26
msgid "_Install Updates"
msgstr "アップデートをインストール(_I)"

#: ../data/glade/UpdateManager.glade.h:27
msgid "_Partial Upgrade"
msgstr "部分的なアップグレード(_P)"

#: ../data/glade/UpdateManager.glade.h:28
msgid "_Upgrade"
msgstr "アップグレード(_U)"

#: ../data/glade/UpdateManager.glade.h:29
msgid "updates"
msgstr "アップデート"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "インストールできるアップデートを表示してインストールします"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "ソフトウェアのアップデート"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "新しいディストリビューションのリリースをチェックする"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "インストール成功の後、インストールウインドウを自動で閉じる"

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "最初に「ようこそ」メッセージを実行する"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"アップデートの自動チェックを無効にすると、チャンネルリストを手動で再読み込みしなければなりません。このオプションを使うと、こういった場合にリマインダの表示"
"を隠すことができます。"

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr "このキーが設定されると、最初の起動時に「ようこそ」メッセージが表示されます。"

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr "このキーを設定すると、インストールに成功してウィンドウが自動的閉じます。"

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr "オンにすると、アップデートの一覧ウィンドウで、バージョンの情報（どのバージョンからどのバージョンへアップデートするか）を表示します。"

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "チャンネルリストの再読み込みを催促する"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "アップデートの詳細を表示"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "アップデートのバージョンを表示"

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "アップデート・マネージャのダイアログサイズを記憶する"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr "変更点と概要を表示する欄の状態を記憶します。"

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "ウィンドウサイズ"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"/etc/update-manager/release-upgrades "
"の状態にしたがって自動的に設定される、廃止予定のキーです。もしディストリビューションの新しいバージョンが利用可能になっていれば、アップグレードするように促"
"します。"

#: ../update-manager:55
msgid "Show version and exit"
msgstr "バージョンを表示して終了"

#: ../update-manager:58
msgid "Check if a new distribution release is available"
msgstr "新しいディストリビューションのリリースが利用可能かチェックする"

#: ../update-manager:61 ../do-release-upgrade:23 ../check-new-release:24
msgid "Check if upgrading to the latest devel release is possible"
msgstr "最新の開発リリースへのアップデートが利用可能かどうかチェックする"

#: ../update-manager:65
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "リリースアップグレードが提案する最新のバージョンを使ってアップグレードを行う"

#: ../update-manager:72
msgid "Do not focus on map when starting"
msgstr "開始時にマップにフォーカスを当てない"

#: ../update-manager:75
msgid "Try to run a dist-upgrade"
msgstr "dist-upgradeを実行する"

#: ../update-manager:79 ../do-release-upgrade:39
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "サンドボックスaufsオーバレイでのテストアップグレード"

#: ../update-manager:98
msgid "Running partial upgrade"
msgstr "部分的なアップグレードを実行"

#: ../do-release-upgrade:27 ../check-new-release:28
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "アップグレーダを使って $distro-proposed から最新のリリースへのアップグレードを試す"

#: ../do-release-upgrade:31
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"特別なアップグレードモードで実行する。\n"
"現在、デスクトップシステムの標準的なアップグレードを行う 'desktop' オプションと、サーバシステム向けの 'server' "
"オプションがサポートされています。"

#: ../do-release-upgrade:37
msgid "Run the specified frontend"
msgstr "特定のフロントエンドで実行"

#: ../do-release-upgrade:43
msgid "Checking for a new ubuntu release"
msgstr "新しいUbuntuのリリースをチェックする"

#: ../do-release-upgrade:50
msgid "No new release found"
msgstr "新しくリリースされたものはありません"

#: ../check-new-release:32
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr "何も出力せずに、新しいリリースがあるなら%sに、なければ%sに戻ります。"

#: ../check-new-release:46
#, c-format
msgid "New release '%s' available."
msgstr "新しいリリース '%s' が利用できます。"

#: ../check-new-release:47
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "'do-release-upgrade' を起動してアップグレードを追加してください"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "未実装のメソッド： %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "ディスクにあるファイル"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb パッケージ"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "不明なパッケージをインストール"

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "パッケージ %s はインストールする必要があります。"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "ステータスファイル中に %i 個の廃止されたエントリがあります。"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "dpkgステータス中に廃止されたエントリがあります"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "廃止されたdpkgステータスエントリ"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug ##279621 for details."
msgstr ""
"アップグレード時に、もしkdelibs4-devがインストールされていればkdelibs5-"
"devをインストールする必要があります。詳細はbugs.launchpad.net の bug ##279621 を参照してください。"

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr ""

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""

#~ msgid "The upgrade tool will guide you through the upgrade process."
#~ msgstr "アップグレード・ツールはアップグレード処理をガイドします。"

#~ msgid "Can't upgrade required meta-packages"
#~ msgstr "要求されたメタパッケージがアップグレードできません"

#~ msgid "Restart the system now [yN] "
#~ msgstr "システムを再起動します [yN] "

#~ msgid "The list of changes is not available"
#~ msgstr "変更点のリストが取得できません。"

#~ msgid "<big><b>Keep your system up-to-date</b></big>"
#~ msgstr "<big><b>システムを最新の状態にする</b></big>"

#~ msgid "Daily"
#~ msgstr "毎日"

#~ msgid "Every two days"
#~ msgstr "2日ごと"

#~ msgid "Weekly"
#~ msgstr "毎週"

#~ msgid "Every two weeks"
#~ msgstr "2週ごと"

#~ msgid "After one week"
#~ msgstr "1週間後"

#~ msgid "After two weeks"
#~ msgstr "2週間後"

#~ msgid "After one month"
#~ msgstr "1ヶ月後"

#~ msgid "Fetching file %li of %li at %sb/s"
#~ msgstr "ダウンロード中: %li / %li 速度 %s バイト/秒"

#~ msgid "%li days %li hours %li minutes"
#~ msgstr "%li 日 %li 時間 %li 分"

#~ msgid "_Check"
#~ msgstr "チェック(_C)"

#~ msgid "Import key"
#~ msgstr "鍵のインポート"

#~ msgid "Error importing selected file"
#~ msgstr "選択したファイルのインポートエラー"

#~ msgid "Error removing the key"
#~ msgstr "鍵削除のエラー"

#~ msgid "Checking for updates"
#~ msgstr "アップデートをチェックしています"

#~ msgid "Every %s days"
#~ msgstr "%s 日ごと"

#~ msgid "After %s days"
#~ msgstr "%s 日後"

#~ msgid "Please enter a name for the disc"
#~ msgstr "ディスク名を入力してください"

#~ msgid "Please insert a disc in the drive:"
#~ msgstr "ディスクをドライブに挿入してください:"

#~ msgid "Authentication"
#~ msgstr "認証"

#~ msgid "_Check for updates automatically:"
#~ msgstr "アップデートを自動的にチェックする(_C):"

#~ msgid "_Import Key File"
#~ msgstr "鍵ファイルのインポート(_I)"

#~ msgid "<b>Comment:</b>"
#~ msgstr "<b>コメント:</b>"

#~ msgid ""
#~ "Binary\n"
#~ "Source"
#~ msgstr ""
#~ "バイナリ\n"
#~ "ソース"

#~ msgid "Scanning CD-ROM"
#~ msgstr "CD-ROM を検査中"

#~ msgid "_Reload"
#~ msgstr "再読込(_R)"

#~ msgid "Officially supported"
#~ msgstr "Officially supported"

#~ msgid "Restricted copyright"
#~ msgstr "Restricted copyright"

#~ msgid "     "
#~ msgstr "     "

#~ msgid ""
#~ "Use special upgrade mode. Available:\n"
#~ "desktop, server"
#~ msgstr ""
#~ "特別なアップグレードモードを使用してください。利用可能なもの:\n"
#~ "desktop, server"

#~ msgid ""
#~ "The list of changes is not available yet.\n"
#~ "Please try again later."
#~ msgstr ""
#~ "変更点のリストはまだ取得できません。\n"
#~ "あとで試してください。"

#~ msgid "_Skip This Step"
#~ msgstr "このステップをスキップ(_S)"

#~ msgid "Asking for confirmation"
#~ msgstr "確認しています"

#~ msgid "<b>Remove %s</b>"
#~ msgstr "<b>削除されるパッケージ: %s</b>"

#~ msgid "Fetch data from the network for the upgrade?"
#~ msgstr "アップグレードをするためにネットワーク経由でデータを取得しますか？"

#~ msgid ""
#~ "The key you selected could not be removed. Please report this as a bug."
#~ msgstr "選択した鍵を削除できませんでした。バグとして報告してください。"

#~ msgid "<p>You have to download a total of %s. "
#~ msgstr "<p>全部で %s 個のパッケージをダウンロードする必要があります。 "

#~ msgid "The selected file may not be a GPG key file or it might be corrupt."
#~ msgstr "選択したファイルはGPG鍵ファイルではないか、壊れている可能性があります。"

#~ msgid "<b>Internet updates</b>"
#~ msgstr "<b>インターネットアップデート</b>"

#~ msgid "Internet Updates"
#~ msgstr "インターネットアップデート"

#~ msgid "<b>URI:</b>"
#~ msgstr "<b>URI:</b>"

#~ msgid "System Update Tool"
#~ msgstr "システムアップデートツール"

#~ msgid "Fetching the upgrades"
#~ msgstr "アップグレードファイルの準備中"

#~ msgid ""
#~ "A unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bugreport."
#~ msgstr ""
#~ "アップグレードを算定中に解決できない問題が発生しました。\n"
#~ "\n"
#~ "このバグを 'update-manager' パッケージに対するものとして、バグレポートに /var/log/dist-upgrade/ "
#~ "以下のファイルを付けて報告してください。"

#~ msgid "Fetching backport of '%s'"
#~ msgstr "'%s\" のバックポートを取得中です"

#~ msgid "Software Channel"
#~ msgstr "ソフトウェアチャンネル"

#~ msgid "(Source Code)"
#~ msgstr "(ソースコード)"

#~ msgid "<b>Automatic updates</b>"
#~ msgstr "<b>自動アップデート</b>"

#~ msgid "<b>Internet</b>"
#~ msgstr "<b>インターネット</b>"

#~ msgid "D_elete downloaded software files:"
#~ msgstr "ダウンロードしたファイルを削除(_E):"

#~ msgid "Download from:"
#~ msgstr "ダウンロード元:"

#~ msgid "Import the public key from a trusted software provider"
#~ msgstr "信頼したソフトウェア供給者の公開鍵をインポートする"

#~ msgid ""
#~ "Only security updates from the official Ubuntu servers will be installed "
#~ "automatically"
#~ msgstr "公式な Ubuntu サーバからのセキュリティアップデートだけ自動的にインストールされます。"

#~ msgid "Restore _Defaults"
#~ msgstr "デフォルトに戻す(_D)"

#~ msgid "Restore the default keys of your distribution"
#~ msgstr "ディストリビューション標準の鍵を元に戻す"

#~ msgid "Software Sources"
#~ msgstr "ソフトウェアソース"

#~ msgid "_Install security updates without confirmation"
#~ msgstr "確認せずにセキュリティアップデートをインストール(_I)"

#~ msgid "<b>Components:</b>"
#~ msgstr "<b>コンポーネント:</b>"

#~ msgid "<b>Distribution:</b>"
#~ msgstr "<b>ディストリビューション:</b>"

#~ msgid "<b>Type:</b>"
#~ msgstr "<b>タイプ:</b>"

#~ msgid "APT line:"
#~ msgstr "APT line:"

#~ msgid "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr ""
#~ "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#~ msgid "Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Ubuntu 6.10 'Edgy Eft'"

#~ msgid "Community maintained"
#~ msgstr "コミュニティによるメンテナンス"

#~ msgid "Restricted software"
#~ msgstr "制限のあるソフトウェア"

#~ msgid "Cdrom with Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Ubuntu 6.10 'Edgy Eft' のCD-ROM"

#~ msgid "Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Ubuntu 6.06 LTS 'Dapper Drake'"

#~ msgid "Canonical supported Open Source software"
#~ msgstr "Canonical によってサポートされるオープンソースソフトウェア"

#~ msgid "Community maintained Open Source software"
#~ msgstr "コミュニティによるメンテナンスされるオープンソースソフトウェア"

#~ msgid "Non-free drivers"
#~ msgstr "フリーではないドライバ"

#~ msgid "Restricted software (Multiverse)"
#~ msgstr "制限されたソフトウェア (Multiuniverse)"

#~ msgid "Cdrom with Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Ubuntu 6.06 LTS 'Dapper Drake' の CD-ROM"

#~ msgid "Ubuntu 5.10 'Breezy Badger'"
#~ msgstr "Ubuntu·5.10·'Breezy·Badger'"

#~ msgid "Ubuntu 5.10 Security Updates"
#~ msgstr "Ubuntu 5.10 セキュリティアップデート"

#~ msgid "Ubuntu 5.10 Updates"
#~ msgstr "Ubuntu 5.10 アップデート"

#~ msgid "Ubuntu 5.10 Backports"
#~ msgstr "Ubuntu 5.10 バックポート"

#~ msgid "Cdrom with Ubuntu 5.04 'Hoary Hedgehog'"
#~ msgstr "CdromとUbuntu 5.04 \"Hoary Hedgehog\""

#~ msgid "Ubuntu 5.04 Security Updates"
#~ msgstr "Ubuntu 5.04 セキュリティアップデート"

#~ msgid "Ubuntu 5.04 Updates"
#~ msgstr "Ubuntu 5.04 アップデート"

#~ msgid "Ubuntu 5.04 Backports"
#~ msgstr "Ubuntu 5.04 バックポート"

#~ msgid "Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Ubuntu 4.10 'Warty Warthog'"

#~ msgid "Community maintained (Universe)"
#~ msgstr "Community maintained (Universe)"

#~ msgid "Non-free (Multiverse)"
#~ msgstr "Non-free (Multiuniverse)"

#~ msgid "Cdrom with Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Ubuntu 4.10 'Warty Warthog' のCD-ROM"

#~ msgid "No longer officially supported"
#~ msgstr "公式にサポートされません"

#~ msgid "Ubuntu 4.10 Security Updates"
#~ msgstr "Ubuntu 4.10 セキュリティアップデート"

#~ msgid "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#~ msgid "Debian 3.1 \"Sarge\""
#~ msgstr "Debian·3.1·\"Sarge\""

#~ msgid "http://security.debian.org/"
#~ msgstr "http://security.debian.org/"

#~ msgid "Debian 3.1 \"Sarge\" Security Updates"
#~ msgstr "Debian·3.1·\"Sarge\"·セキュリティアップデート"

#~ msgid "Debian \"Etch\" (testing)"
#~ msgstr "Debian·\"Etch\"·(testing)"

#~ msgid "http://http.us.debian.org/debian/"
#~ msgstr "http://http.us.debian.org/debian/"

#~ msgid "Debian \"Sid\" (unstable)"
#~ msgstr "Debian·\"Sid\"·(unstable)"

#~ msgid "DFSG-compatible Software with Non-Free Dependencies"
#~ msgstr "非フリーな依存関係のあるDFSG適合ソフトウェア"

#~ msgid "Non-DFSG-compatible Software"
#~ msgstr "DFSGに適合しないソフトウェア"

#~ msgid "Downloading file %li of %li with unknown speed"
#~ msgstr "ダウンロード中: 速度不明で %li のうち %li"

#~ msgid "Cancel _Download"
#~ msgstr "ダウンロードをキャンセル(_D)"

#~ msgid "Could not find any upgrades"
#~ msgstr "アップグレードが見つかりません"

#~ msgid "Your system has already been upgraded."
#~ msgstr "システムはすでに最新の状態です。"

#~ msgid "Cannot install all available updates"
#~ msgstr "すべてのアップデートをインストールすることができません"

#~ msgid ""
#~ "<big><b>Examining your system</b></big>\n"
#~ "\n"
#~ "Software updates correct errors, eliminate security vulnerabilities and "
#~ "provide new features."
#~ msgstr ""
#~ "<big><b>システムを解析中です</b></big>\n"
#~ "\n"
#~ "ソフトウェアアップデートはエラーを修正し、セキュリティホールを修正し、新機能を提供します。"

#~ msgid ""
#~ "Some updates require the removal of further software. Use the function "
#~ "\"Mark All Upgrades\" of the package manager \"Synaptic\" or run \"sudo apt-"
#~ "get dist-upgrade\" in a terminal to update your system completely."
#~ msgstr ""
#~ "ほかのソフトを削除しなければならないアップデートがあります。完全にアップグレードするためにはパッケージマネージャ \"Synaptic\" の "
#~ "\"すべてのアップグレードににマーク\"機能を使うか。端末から \"sudo apt-get dist-upgrade\" を実行してください。"

#~ msgid "The following updates will be skipped:"
#~ msgstr "これらのパッケージはアップグレードされません:"

#~ msgid "Download is complete"
#~ msgstr "ダウンロードが完了しました"

#~ msgid "The upgrade aborts now. Please report this bug."
#~ msgstr "アップグレードが中断しました。不具合を報告してください。"

#~ msgid "Upgrading Ubuntu"
#~ msgstr "Ubuntu のアップグレード中"

#~ msgid "Hide details"
#~ msgstr "詳細を隠す"

#~ msgid "Show details"
#~ msgstr "詳細を表示"

#~ msgid "Only one software management tool is allowed to run at the same time"
#~ msgstr "ソフトウェアマネージメントツールは同時に1つしか実行することができません"

#~ msgid ""
#~ "Please close the other application e.g. 'aptitude' or 'Synaptic' first."
#~ msgstr "まず 'aptitude' または 'Synaptic' など、ほかのアプリケーションを終了してください。"

#~ msgid "<b>Channels</b>"
#~ msgstr "<b>チャンネル</b>"

#~ msgid "<b>Keys</b>"
#~ msgstr "<b>鍵</b>"

#~ msgid "Installation Media"
#~ msgstr "インストールメディア"

#~ msgid "Software Preferences"
#~ msgstr "ソフトウェアの設定"

#~ msgid "<b>Channel</b>"
#~ msgstr "<b>チャンネル</b>"

#~ msgid "<b>Components</b>"
#~ msgstr "<b>コンポーネント</b>"

#~ msgid "Add Channel"
#~ msgstr "チャンネルを追加"

#~ msgid "_Add Channel"
#~ msgid_plural "_Add Channels"
#~ msgstr[0] "チャンネルを追加(_A)"

#~ msgid "_Custom"
#~ msgstr "カスタム(C)"

#~ msgid "Ubuntu 6.06 LTS Security Updates"
#~ msgstr "Ubuntu 6.06 LTS セキュリティアップデート"

#~ msgid "Ubuntu 6.06 LTS Updates"
#~ msgstr "Ubuntu 6.06 LTS アップデート"

#~ msgid "Ubuntu 6.06 LTS Backports"
#~ msgstr "Ubuntu 6.06 LTS バックポート"

#~ msgid "<b>Sections</b>"
#~ msgstr "<b>セクション:</b>"

#~ msgid "<b>Sections:</b>"
#~ msgstr "<b>セクション:</b>"

#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Downloading changes</span>\n"
#~ "\n"
#~ "Need to get the changes from the central server"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">変更点を取得中</span>\n"
#~ "\n"
#~ "中央サーバから変更点を取得する必要があります"

#~ msgid "Show available updates and choose which to install"
#~ msgstr "アップデート可能なファイルの表示とインストール"

#~ msgid "Edit software sources and settings"
#~ msgstr "ソフトウェアのソースと設定を編集"

#~ msgid "<b>Repository</b>"
#~ msgstr "<b>リポジトリ</b>"

#~ msgid "<b>Temporary files</b>"
#~ msgstr "<b>一時ファイル</b>"

#~ msgid "<b>User Interface</b>"
#~ msgstr "ユーザインターフェース"

#~ msgid ""
#~ "<big><b>Authentication keys</b></big>\n"
#~ "\n"
#~ "You can add and remove authentication keys in this dialog. A key makes it "
#~ "possible to verify the integrity of the software you download."
#~ msgstr ""
#~ "<big><b>認証鍵</b></big>\n"
#~ "\n"
#~ "認証鍵の追加と削除が行えます。認証鍵によりダウンロードしたソフトウェアが完全なものか確認することができます。"

#~ msgid ""
#~ "Add a new key file to the trusted keyring. Make sure that you received the "
#~ "key over a secure channel and that you trust the owner. "
#~ msgstr "新しい鍵ファイルを信頼されたキーリングに追加します。セキュアなチャンネル経由で鍵を取得し、信頼される持ち主のものか確認してください。 "

#~ msgid "Automatically clean _temporary packages files"
#~ msgstr "一時ファイルを自動的に削除する(_T)"

#~ msgid "Clean interval in days: "
#~ msgstr "削除する間隔(日): "

#~ msgid "Delete _old packages in the package cache"
#~ msgstr "パッケージキャッシュにある古いパッケージを削除する(_O)"

#~ msgid "Edit Repository..."
#~ msgstr "リポジトリの編集..."

#~ msgid "Maximum age in days:"
#~ msgstr "最長の保存期間(日):"

#~ msgid "Maximum size in MB:"
#~ msgstr "最大量(MB):"

#~ msgid ""
#~ "Restore the default keys shipped with the distribution. This will not change "
#~ "user installed keys."
#~ msgstr "ディストリビューション付属のデフォルトの鍵を元に戻します。この変更によりユーザが追加した鍵が失われることはありません。"

#~ msgid "Set _maximum size for the package cache"
#~ msgstr "パッケージキャッシュの最大量を設定する(_M)"

#~ msgid "Show disabled software sources"
#~ msgstr "無効なソフトウェア取得元を表示"

#~ msgid "Update interval in days: "
#~ msgstr "アップデートする間隔(日): "

#~ msgid "_Add Repository"
#~ msgstr "リポジトリの追加(_A)"

#~ msgid "_Download upgradable packages"
#~ msgstr "アップグレード可能なパッケージを取得する(_D)"

#~ msgid ""
#~ "<big><b>Available Updates</b></big>\n"
#~ "\n"
#~ "The following packages are found to be upgradable. You can upgrade them by "
#~ "using the Install button."
#~ msgstr ""
#~ "<big><b>アップデートがあります</b></big>\n"
#~ "\n"
#~ "以下のパッケージがアップグレード可能です。インストールボタンを押すとこれらのパッケージがインストールされます。"

#~ msgid "You need to be root to run this program"
#~ msgstr "rootで実行してください"

#~ msgid "Binary"
#~ msgstr "バイナリ"

#~ msgid "Non-free software"
#~ msgstr "非フリーソフトウェア"

#~ msgid "Ubuntu Archive Automatic Signing Key <ftpmaster@ubuntu.com>"
#~ msgstr "Ubuntu Archive Automatic Signing Key <ftpmaster@ubuntu.com>"

#~ msgid "Ubuntu CD Image Automatic Signing Key <cdimage@ubuntu.com>"
#~ msgstr "Ubuntu CD Image Automatic Signing Key <cdimage@ubuntu.com>"

#~ msgid "Choose a key-file"
#~ msgstr "キーファイルを選択"

#~ msgid "The updates are being applied."
#~ msgstr "アップデートされました。"

#~ msgid "Updating package list..."
#~ msgstr "アップデートされるパッケージのリスト..."

#~ msgid "There are no updates available."
#~ msgstr "アップデートするものはありません。"

#~ msgid ""
#~ "Please upgrade to a newer version of Ubuntu Linux. The version you are "
#~ "running will no longer get security fixes or other critical updates. Please "
#~ "see http://www.ubuntulinux.org for upgrade information."
#~ msgstr ""
#~ "新しいUbuntu "
#~ "Linuxにアップグレードしてください。現在お使いのシステムにはセキュリティフィクスや危急のアップデートはすでに提供されていません。アップグレードに関する"
#~ "情報は http://www.ubuntulinux.org/ を見てください。"

#~ msgid "There is a new release of Ubuntu available!"
#~ msgstr "Ubuntuの新しいリリース版があります！"

#~ msgid ""
#~ "A new release with the codename '%s' is available. Please see "
#~ "http://www.ubuntulinux.org/ for upgrade instructions."
#~ msgstr ""
#~ "コードネーム '%s' という新しいリリース版があります。アップグレードするために http://www.ubuntulinux.org/ "
#~ "をご覧ください。"

#~ msgid "Never show this message again"
#~ msgstr "このメッセージを二度と表示しない"

#~ msgid "Changes not found, the server may not be updated yet."
#~ msgstr "変更点は見つかりませんでした。サーバはまだアップデートされていないようです。"

#~ msgid "Ubuntu 6.06 LTS"
#~ msgstr "Ubuntu 6.06 LTS"

#~ msgid "Some software no longer officially supported"
#~ msgstr "いくつかのソフトウェアはもう公式にサポートされません"

#~ msgid "About %li seconds remaining"
#~ msgstr "およそ残り %li 秒"

#~ msgid "Source Code"
#~ msgstr "ソースコード"

#~ msgid ""
#~ "<big><b>Error scanning the CD</b></big>\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "<big><b>CDスキャンエラー</b></big>\n"
#~ "\n"
#~ "%s"

#~ msgid "Add Cdrom"
#~ msgstr "CD-ROM の追加"

#~ msgid "Source code"
#~ msgstr "ソースコード"

#~ msgid "_Download updates automatically, but do not install them"
#~ msgstr "インストールはしないが、アップデートを自動的にダウンロード"

#~ msgid "Run in a special upgrade mode"
#~ msgstr "特別なアップグレードモードで実行"

#~ msgid "None"
#~ msgstr "なし"

#~ msgid "%s updates"
#~ msgstr "%s アップデート"

#~ msgid "%s (%s)"
#~ msgstr "%s (%s)"

#~ msgid "Nearest server"
#~ msgstr "最も近いサーバ"

#~ msgid "Active"
#~ msgstr "有効"

#~ msgid ""
#~ "The upgrade can use the network to check the latest updates and to fetch "
#~ "packages that are not on the current CD.\n"
#~ "If you have fast or inexpensive network access you should answer 'Yes' here. "
#~ "If networking is expensive for you choose 'No'."
#~ msgstr ""
#~ "アップグレードは、ネットワークを利用して最新のアップデートを確認し、現在の CD に無いパッケージを取得することができます。\n"
#~ "高速、または安価なネットワークアクセスがある場合は、ここで 'はい' を選んでください。そのようなネットワークが無い場合は 'いいえ' を選んでください。"

#~ msgid ""
#~ "The upgrade aborts now. Your system could be in an unusable state. A "
#~ "recovery was run (dpkg --configure -a).\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bugreport."
#~ msgstr ""
#~ "アップグレードを中断しました。システムが使用できない状態になっている可能性があります。ただいま修正を実行中です (dpkg --configure -"
#~ "a)。\n"
#~ "\n"
#~ "このバグを 'update-manager' パッケージのバグとして報告して下さい。その際、バグ報告に /var/log/dist-upgrade/ "
#~ "中のファイルを含めて下さい。"

#~ msgid "Changes and description of the update"
#~ msgstr "アップデートの変更点と詳細"

#~ msgid "<b>CDROM/DVD</b>"
#~ msgstr "<b>CDROM/DVD</b>"

#~ msgid "Third Party"
#~ msgstr "サードパーティ"

#~ msgid "Proprietary drivers for devices"
#~ msgstr "デバイス用のプロプライエタリなドライバ"

#~ msgid "Proprietary drivers for devices "
#~ msgstr "デバイス用のプロプライエタリなドライバ "

#~ msgid "Backported updates"
#~ msgstr "バックポートされたアップデート"

#~ msgid "Oficially supported"
#~ msgstr "公式サポート"

#~ msgid "Reload the latest information about updates"
#~ msgstr "アップデートに関する最新情報を再読み込み"

#~ msgid ""
#~ "While scanning your repository information no valid entry for the upgrade "
#~ "was found.\n"
#~ msgstr "アップグレードするためにリポジトリ情報を調べている際、正しくないエントリがみつかりました。\n"

#~ msgid "<b>Packages to install:</b>"
#~ msgstr "<b>インストールするパッケージ:</b>"

#~ msgid "A_uthentication"
#~ msgstr "認証(_U)"

#~ msgid ""
#~ "This means that some dependencies of the installed packages are not "
#~ "satisfied. Please use \"Synaptic\" or \"apt-get\" to fix the situation."
#~ msgstr ""
#~ "インストールされたパッケージの依存性が満たせないようです。\"Synaptic\" または \"apt-get\" を使用して修正してください。"

#~ msgid "It is not possible to upgrade all packages."
#~ msgstr "全てのパッケージをアップグレードすることは不可能です。"

#~ msgid ""
#~ "This means that besides the actual upgrade of the packages some further "
#~ "action (such as installing or removing packages) is required. Please use "
#~ "Synaptic \"Smart Upgrade\" or \"apt-get dist-upgrade\" to fix the situation."
#~ msgstr ""
#~ "パッケージのアップグレードの他にパッケージのインストールや削除などの別の対処がいるようです。Synaptic \"Smart "
#~ "Upgrade\"か\"apt-get dist-upgrade\"を実行して問題を修正してください。"

#~ msgid ""
#~ "You can run only one package management application at the same time. Please "
#~ "close this other application first."
#~ msgstr "同時にひとつのパッケージマネージャしか起動できません。先に他のアプリケーションマネージャを終了してください。"

#~ msgid "Initializing and getting list of updates..."
#~ msgstr "アップデートリストを取得中..."

#~ msgid "Ubuntu Update Manager"
#~ msgstr "Ubuntuアップデートマネージャ"

#~ msgid "CD"
#~ msgstr "CD"

#~ msgid "_Download updates in the background, but do not install them"
#~ msgstr "アップデートをダウンロードはするが、インストールはしない(_D)"

#~ msgid "Repositories changed"
#~ msgstr "リポジトリが変更されました"

#~ msgid ""
#~ "You need to reload the package list from the servers for your changes to "
#~ "take effect. Do you want to do this now?"
#~ msgstr "変更点を有効にするためにパッケージリストをサーバから再取得する必要があります。今すぐ実行しますか?"

#~ msgid "Remove obsolete Packages?"
#~ msgstr "古いパッケージ削除しますか？"

#~ msgid ""
#~ "Reload the package information from the server. \n"
#~ "\n"
#~ "If you have a permanent internet connection this is done automatically. If "
#~ "you are behind an internet connection that needs to be started by hand (e.g. "
#~ "a modem) you should use this button so that update-manager knows about new "
#~ "updates."
#~ msgstr ""
#~ "サーバからパッケージ情報を読み込み直します。\n"
#~ "\n"
#~ "ずっとインターネットに接続されたままなら、自動的に行います。アナログモデムなどでそうではないなら、アップデートマネージャに新しいアップデートがあるかどうか"
#~ "を知らせるため、このボタンを使用して手動で行う必要があります。"

#~ msgid ""
#~ "<span weight=\"bold\" size=\"x-large\">Upgrading to Ubuntu \"Dapper\" "
#~ "6.04</span>"
#~ msgstr ""
#~ "<span·weight=\"bold\"·size=\"x-large\">Ubuntu·\"Dapper\"·6.04 "
#~ "にアップグレード中</span>"

#~ msgid ""
#~ "<big><b>Checking for available updates</b></big>\n"
#~ "\n"
#~ "Software updates can correct errors, eliminate security vulnerabilities, and "
#~ "provide new features to you."
#~ msgstr ""
#~ "<big><b>アップデートをチェック中</b></big>\n"
#~ "\n"
#~ "アップデートによってソフトウェアの問題を修正し、セキュリティホールを除去し、新機能を追加します。"

#~ msgid "Ubuntu 6.04 'Dapper Drake'"
#~ msgstr "Ubuntu·6.04·'Dapper·Drake'"

#~ msgid "Ubuntu 6.04 Security Updates"
#~ msgstr "Ubuntu 6.04 セキュリティアップデート"

#~ msgid "Ubuntu 6.04 Updates"
#~ msgstr "Ubuntu 6.04 アップデート"

#~ msgid "Ubuntu 6.04 Backports"
#~ msgstr "Ubuntu 6.04 バックポート"

#~ msgid ""
#~ "There is not enough free space on your system to download the required "
#~ "pacakges. Please free some space before trying again with e.g. 'sudo apt-get "
#~ "clean'"
#~ msgstr ""
#~ "システムに要求されたパッケージをダウンロードするだけの十分な空き容量がありません。再び試行する前に 'sudo apt-get clean' "
#~ "などで空き容量を確保してください"

#~ msgid ""
#~ "Some problem occured during the fetching of the packages. This is most "
#~ "likely a network problem. Please check your network and try again. "
#~ msgstr "パッケージを取得中になんらかのエラーが発生しました。おそらくネットワークの問題です。ネットワークを確認して再試行してください。 "

#~ msgid ""
#~ "%s packages are going to be removed.\n"
#~ "%s packages are going to be newly installed.\n"
#~ "%s packages are going to be upgraded.\n"
#~ "\n"
#~ "%s needs to be fetched"
#~ msgstr ""
#~ "%s·つのパッケージが削除されます。\n"
#~ "%s·つのパッケージが新規インストールされます。\n"
#~ "%s·つのパッケージがアップグレードされます。\n"
#~ "\n"
#~ "%s·つのパッケージを取得します"

#~ msgid "To be installed: %s"
#~ msgstr "インストールされるパッケージ:·%s"

#~ msgid "Are you sure you want cancel?"
#~ msgstr "本当にキャンセルしますか？"

#~ msgid ""
#~ "Canceling during a upgrade can leave the system in a unstable state. It is "
#~ "strongly adviced to continue the operation. "
#~ msgstr "アップグレード中にキャンセルすると、システムが不安定な状態になります。動作を継続することを強く忠告します。 "
